flink-runtime/src/test/java/org/apache/flink/runtime/state/v2/AbstractAggregatingStateTest.java

```diff
@@ -67,6 +67,12 @@ public Integer merge(Integer a, Integer b) {
         validateRequestRun(state, StateRequestType.AGGREGATING_GET, null, 0);
 
         state.asyncAdd(1);
-        validateRequestRun(state, StateRequestType.AGGREGATING_ADD, 1, 0);
+        validateRequestRun(state, StateRequestType.AGGREGATING_GET, null, 1);
+        validateRequestRun(state, StateRequestType.AGGREGATING_PUT, 2, 0);
+
+        state.asyncAdd(5);
+        validateRequestRun(state, StateRequestType.AGGREGATING_GET, null, false);
```

Comment:
```suggestion
        validateRequestRun(state, StateRequestType.AGGREGATING_GET, null, 1);
```

And remove this function?
```
<IN> void validateRequestRun(
            @Nullable State state, StateRequestType type, @Nullable IN payload, boolean isEnd)
```

Reply:
done